src/components/Table/sortRows.ts

```diff
@@ -11,6 +11,9 @@ export function sortRows<R extends Kinded>(
 ): GridDataRow<R>[] {
   const sorted = sortBatch(columns, rows, sortState, caseSensitive);
   // Recursively sort child rows
+
+  // check for persistent columns her move to top
```

Comment:
Fwiw my 1st guess would be to just "sort twice".

I.e. somehow get ahold of the `persistent` key in here (from `sortState`?) and, if it's set, do a 1st pass sort on `sortBatch` by somehow tell it to "actually use the persistent column", and then do a 2nd pass / 2nd `sortBatch` to sort on the current.

Like this line:

```
  const [value, direction] = sortState;
  const column = columns[value];
```

Would on the 1st pass use the persistent column index for `value`, and on the 2nd pass use actual `value` / current column index.

That will probably be the least terrible way of making this sort, albeit at the "cost" of doing two passes.

A fancier way of doing it would be to only do 1 pass, but teach the sort to use `persistent` first, i.e. around here:

```
    const v1 = sortValue(applyRowFn(column, a, {} as any, 0), caseSensitive);
    const v2 = sortValue(applyRowFn(column, b, {} as any, 0), caseSensitive);
```

Add something like:

```
    const v1p = sortValue(applyRowFn(persistentColumn, a, {} as any, 0), caseSensitive);
    const v2p = sortValue(applyRowFn(persistentColumn, b, {} as any, 0), caseSensitive);
```

And then work `v1p` and `v2p` into the various expressions below that. Which is doable but seems hard.